app/controllers/objects_controller.rb

```diff
@@ -7,13 +7,13 @@
 #  (Note: methods will eventually be ported from sdr-services-app)
 class ObjectsController < ApplicationController
   # return the PreservedObject model for the druid (supplied with druid: prefix)
```

Comment:
*does* it require the druid prefix?  I thought we took it either way?

Reply:
Addressed here: https://github.com/sul-dlss/preservation_catalog/pull/1408 Resolving.